src/firebase/client.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Configuration to use the database on the client side.
+ */
+
+import { initializeApp } from "firebase/app";
+
+const firebaseConfig = {
+  apiKey: "AIzaSyCJKH1IpkXR7RcuVY3kWwz8D3v2Rj1zpuY",
```

Comment:
lets hide this first

Reply:
I stored them in environment variables. For some funky reason they had to be prefixed with PUBLIC, but they're now hidden 🤓